pipeline/outputs/s3.md

```diff
@@ -49,6 +49,13 @@ See [here](https://github.com/fluent/fluent-bit-docs/tree/43c4fe134611da471e706b
 | external\_id                     | Specify an external ID for the STS API, can be used with the role\_arn parameter if your role requires an external ID.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | None                                    |
+| parquet.compression             | Compression type for parquet. 'uncompressed', 'snappy', 'gzip', 'zstd' are the supported values by default. 'lzo', 'brotli', 'lz4' are not supported for now.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | SNAPPY                                  |
```

Comment:
lowercase `snappy`?

Reply:
No. Columnify does not accept lower cases of compression methods.

Reply:
OK then uppercase the values in the description and let's add a note saying it must be uppercase. Otherwise people will assume you can use whatever like the rest of Fluent Bit config.

Reply:
I noticed that the converter which converts from lower case to upper case is already implemented in out_s3. So, I added descriptions for both of lower case and upper case should be accepted this parameter.